modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormFieldTemplateContextContributorUtil.java

```diff
@@ -63,6 +67,51 @@ public static Map<String, Object> getLocaleMap(Locale defaultLocale) {
 		).build();
 	}
 
+	public static List<Map<String, Object>> getOptions(
+		DDMFormField ddmFormField,
+		ListTypeEntryLocalService listTypeEntryLocalService) {
+
+		List<Map<String, Object>> options = new ArrayList<>();
+
+		DDMFormFieldOptions ddmFormFieldOptions =
+			(DDMFormFieldOptions)ddmFormField.getProperty("options");
```

Comment:
I believe the correct way to declare these variables is as follows:
```
DDMFormFieldOptions ddmFormFieldOptions =
	(DDMFormFieldOptions)ddmFormField.getProperty("options");
List<Map<String, Object>> options = new ArrayList<>();
```

it is the way it was in [MultiselectPicklistDDMFormFieldTemplateContextContributor.java](https://github.com/liferay-bpm/liferay-portal/pull/4349/commits/c169608de592a2dbea1f6f120adc8165f8f81903?diff=unified&w=0#diff-2f1f687d0ce52863263daa4f2580ea2e82406c54fcd15f241fe1908cf14f7102)